### PR TITLE
Add sogga

### PR DESCRIPTION
**Pull request by Indikativ**
https://discord.com/channels/777536797601300490/777539157903671316/843845887813222421

ist **nicht** getestet
was man testen sollte: die 3 bilder/gifs, line 15-17

wenn ich das teste dauert das wieder 5 stunden

### CODE REVIEW

File: src/main/kotlin/net/axay/blaubot/commands/implementation/Sogga.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        "https://media1.tenor.com/images/4c771b4cd0aaedbedb6e87cd6feaaf3d/tenor.gif",
+        "https://static.wikia.nocookie.net/floppapedia-revamped/images/6/6c/Obraz_2021-02-15_135626.webp/revision/latest/top-crop/width/360/height/450?cb=20210224164127"
+    )
+
+    override suspend fun execute(interaction: Interaction, command: InteractionCommand) {
+        interaction.ackowledgePublic().followUp {
+            embed {
+                title = "Sogga"
+                image = soggaImages.random()
+                description = "Sogga is an extension meme of Floppa. Both charakters are often used in the same way"
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/blaubot/commands/implementation/Sogga.kt]: "charakters" -> "characters"

and please add a dot at the end of the sentence

File: src/main/kotlin/net/axay/blaubot/commands/implementation/Sogga.kt [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            }
+        }
+    }
+
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/blaubot/commands/implementation/Sogga.kt]: remove the two blank lines

File: src/main/kotlin/net/axay/blaubot/commands/implementation/Sogga.kt [hunk 3]
```diff
@@ -0,0 +1,31 @@
+package net.axay.blaubot.commands.implementation
+
+import dev.kord.common.annotation.KordPreview
+import dev.kord.core.behavior.interaction.followUp
+import dev.kord.core.entity.interaction.Interaction
+import dev.kord.core.entity.interaction.InteractionCommand
+import net.axay.blaubot.commands.api.SlashCommand
+
+@KordPreview
+object Sogga : SlashCommand (
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/blaubot/commands/implementation/Sogga.kt]: remove space after `SlashCommand`

File: src/main/kotlin/net/axay/blaubot/commands/implementation/Sogga.kt [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    "sogga",
+    "Shows sogga to you"
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/blaubot/commands/implementation/Sogga.kt]: add trailing comma

**Review by jakobkmar — CHANGES_REQUESTED** [review]

**Review by jakobkmar — APPROVED** [review]

**Comment by jakobkmar** [comment]
build succeeded, the update will be available soon

**Comment by Indikativ** [comment]
my first pull request got merged :|

**Pull request merged**